client/apis/volumegroupsnapshot/v1beta1/types.go

```diff
@@ -341,24 +347,24 @@ type VolumeGroupSnapshotContentStatus struct {
 	// On Unix, the command date +%s%N returns the current time in nanoseconds
 	// since 1970-01-01 00:00:00 UTC.
 	// +optional
-	CreationTime *int64 `json:"creationTime,omitempty" protobuf:"varint,2,opt,name=creationTime"`
+	CreationTime *int64 `json:"creationTime,omitempty" protobuf:"varint,3,opt,name=creationTime"`
```

Comment:
This is a breaking change in the gRPC protocol, has that been considered? Existing indexes should ideally not be modified to keep it backwards compatible.

Reply:
You are right. I can definitely use a new field index for the new field, and keep the ones we already use steady.

Reply:
Thank you for spotting this. I updated the commit and fixed it.